### PR TITLE
html first writing for homepage

### PR DESCRIPTION
**Pull request by QTenQC**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,81 @@
...
-<html>
-  <head>
-  </head>
-  <body>
-    <div>
-    </div>
-  </body>
+<html lang = en>
+<head>
+	<meta charset="utf-8">
```
Comment (TheActualWalko) [inline index.html]: Need to set editor to use spaces instead of tabs!

File: index.html [hunk 2]
```diff
@@ -1,9 +1,81 @@
 <!DOCTYPE html>
-<html>
-  <head>
-  </head>
-  <body>
-    <div>
-    </div>
-  </body>
+<html lang = en>
```
Comment (TheActualWalko) [inline index.html]: For element attributes, always use double quotes around the value, and never use spaces around the `=`. For example, here we'd like to see
`<html lang="en">`

File: index.html [hunk 3]
```diff
@@ -1,9 +1,81 @@
...
+<head>
+	<meta charset="utf-8">
+	<title>Rocket Science</title>
+</head>
+<body>
+	<div id = "branding">
```
Comment (TheActualWalko) [inline index.html]: See above - no spaces around the `=` pls!

File: index.html [hunk 4]
```diff
@@ -1,9 +1,81 @@
 <!DOCTYPE html>
-<html>
-  <head>
-  </head>
-  <body>
-    <div>
-    </div>
-  </body>
+<html lang = en>
+<head>
```
Comment (TheActualWalko) [inline index.html]: Please indent everything inside the `<html>` block as well!

File: index.html [hunk 5]
```diff
@@ -1,9 +1,81 @@
...
+			<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut.</p>
+			<img src = "" alt = "email"/> <img src = "" alt = "link"/> <img src = "" alt = "linkedIn"/>
+		</div>
+
+	</div>
+
+
+
+	<footer>
+		&copy; 2017, Rocket Science
```
Comment (TheActualWalko) [inline index.html]: 🅰️ +

**Review by TheActualWalko — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -1,81 +1,81 @@
...
+    <div id="branding">
+      <img src="" alt="logo"/>
+      <h1>Rocket Science</h1>
+      <h2>Software Craftsmanship</h2>
+    </div>
 
-	<div id = "introText">
-		<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
-	</div>
+    <div id="introText">
```
Comment (TheActualWalko) [inline index.html]: we'll use camel case in javascript quite a bit - but for raw HTML and CSS i prefer to use the form `intro-text` and `main-content`. Old browsers didn't distinguish between upper- and lower-case characters and as a result it has become standard to use dashes in classnames!

File: index.html [hunk 7]
```diff
@@ -1,81 +1,81 @@
...
-			<h4>Job title description</h4>
-			<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut.</p>
-			<img src = "" alt = "email"/> <img src = "" alt = "link"/> <img src = "" alt = "linkedIn"/>
-		</div>
+      <div class="teamMember">
+        <img src="" alt="headshot"/>
+        <h3>Name</h3>
+        <h4>Job title description</h4>
+        <p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut.</p>
+        <img src="" alt="email"/> <img src="" alt="link"/> <img src="" alt="linkedIn"/>
```
Comment (TheActualWalko) [inline index.html]: alt text for all of these should be in title case; treat it as though it was a heading on part of the document. `Email` and `LinkedIn`!

File: index.html [hunk 8]
```diff
@@ -1,81 +1,81 @@
...
 
-	<div id = "introText">
-		<p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
-	</div>
+    <div id="introText">
+      <p>Lorem ipsum dolor sit amet, quo eu sanctus omnesque, an quas splendide pro, maiorum fabellas per et. Vix et prima bonorum adipiscing, ius zril inermis ex. Sea ne omnis modus tincidunt, eos an cetero singulis torquatos. Et erat pertinacia cum, instructior reprehendunt vel ut. Ex eligendi insolens sapientem per, duo id tale sumo propriae, ut falli inermis cum.</p>
+    </div>
 
-	<hr>
+    <hr>
```
Comment (TheActualWalko) [inline index.html]: lets scrap this guy - `<hr>` is an oldskool presentational element that we shouldn't need to use any more. We'll accomplish this by adding a `border-bottom` to the `<div>` above (or something similar)!

**Review by TheActualWalko — CHANGES_REQUESTED** [review]
looking great! almost there!

File: index.html (hunk 8 above)
Comment (TheActualWalko) [reply index.html]: reason for this is to keep ALL of our styling in css and use HTML for document structure only - a horizontal rule or `<hr>` is a stylistic touch, and should be expressed in CSS to keep things tidy.

File: index.html (hunk 6 above)
Comment (QTenQC) [reply index.html]: okay cool beans! commenting to show I'm learning how this all works! :)

File: index.html [hunk 9]
```diff
@@ -1,9 +1,79 @@
 <!DOCTYPE html>
-<html>
+<html lang=en>
```
Comment (TheActualWalko) [inline index.html]: quotes around `en` and we're done!!!!

**Review by TheActualWalko — APPROVED** [review]
BOOM

**Comment by QTenQC** [comment]
c'est dope le!!

**Pull request merged**